test_conformance/spirv_new/spirv_asm/op_spec_constant_uint_simple.spvasm32

```diff
@@ -0,0 +1,74 @@
+; SPIR-V
+; Version: 1.0
+; Generator: Khronos LLVM/SPIR-V Translator; 14
+; Bound: 38
+; Schema: 0
+               OpCapability Addresses
+               OpCapability Linkage
+               OpCapability Kernel
+               OpCapability Int8
+          %1 = OpExtInstImport "OpenCL.std"
+               OpMemoryModel Physical32 OpenCL
+               OpEntryPoint Kernel %9 "spec_const_kernel" %__spirv_BuiltInGlobalInvocationId
+         %36 = OpString "kernel_arg_type.spec_const_kernel.uint*,uint*,"
+               OpSource OpenCL_C 102000
+               OpName %__spirv_BuiltInGlobalInvocationId "__spirv_BuiltInGlobalInvocationId"
+               OpName %numbers_buffer "numbers_buffer"
+               OpName %spec_constant_value "spec_constant_value"
+               OpName %entry "entry"
+               OpName %numbers_buffer_addr "numbers_buffer.addr"
+               OpName %spec_constant_value_addr "spec_constant_value.addr"
+               OpName %id "id"
+               OpName %call "call"
+               OpName %arrayidx "arrayidx"
+               OpName %add "add"
+               OpName %arrayidx1 "arrayidx1"
+               OpDecorate %37 Alignment 4
+         %37 = OpDecorationGroup
+               OpDecorate %__spirv_BuiltInGlobalInvocationId BuiltIn GlobalInvocationId
+               OpDecorate %__spirv_BuiltInGlobalInvocationId Constant
+               OpDecorate %__spirv_BuiltInGlobalInvocationId LinkageAttributes "__spirv_BuiltInGlobalInvocationId" Import
+               OpGroupDecorate %37 %numbers_buffer_addr %spec_constant_value_addr %id
+       %uint = OpTypeInt 32 0
+      %uchar = OpTypeInt 8 0
+     %uint_0 = OpConstant %uint 0
```

Comment:
I'm guessing you intended this to be a spec constant. It would be good to understand why the test passed (it's not really fit for purpose if it lets that through).

Reply:
Sorry this is my mistake. The test was failing. I fixed 64 bit version but forgot about 32. Thanks for catching this.

Reply:
@kpet Wondering what your views on this are now. It's a lot better than it was, although there's still some room for improvement. Would it be okay to address this separately, once we've removing the binaries, do you think?

Reply:
@StuartDBrady Assuming you intended to reply to that comment https://github.com/KhronosGroup/OpenCL-CTS/pull/719#pullrequestreview-412082959 and not this thread, I think we should indeed address that after removing the binaries. Getting forward progress on this PR is definitely more important IMHO.